docker-compose.yml

```
@@ -178,6 +178,20 @@ services:
     volumes:
       - "lambdas_server_storage:/app/storage/lambdas"
 
+  lamb2:
+    image: quay.io/decentraland/lamb2:${LAMB2_DOCKER_TAG:-latest}
+    working_dir: /app
+    restart: always
+    environment:
+      - HTTP_SERVER_PORT=7272
+      - CONTENT_SERVER_ADDRESS=https://peer.decentraland.org/content
```

Should this point to the internal content server?
```suggestion
      - CONTENT_SERVER_ADDRESS=http://content
```

---

Yessss, good catch agus. Thanks

---

Also secured metrics endpoints plz 🙏